Neos.ContentRepository.Core/Classes/Feature/NodeReferencing/Dto/NodeReferenceToWrite.php

```diff
@@ -26,32 +27,37 @@
  * (this is validated in the command handler).
  * @api used as part of commands
  */
-final readonly class NodeReferenceToWrite implements \JsonSerializable
+final readonly class NodeReferenceToWrite
```

Comment:
thanks for improving the api for setting references ill look into it how this new way is to be used instead and what is to be migrated for Neos 9 users...

While fine tuning the references api id like to also note that it often came to my mind that it didnt seem to have gotten the same love as setting properties. I think both setting references and properties should feel similar and not be totally a different thing.